samples/corda/corda-simple-application/clients/src/main/kotlin/com/cordaSimpleApplication/client/AssetStateManager.kt

```diff
@@ -0,0 +1,284 @@
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.cordaSimpleApplication.client
+
+import com.cordaSimpleApplication.flow.AssetFlow.IssueAssetState
+import com.cordaSimpleApplication.flow.AssetFlow.GetStatesByTokenType
+import com.cordaSimpleApplication.flow.AssetFlow.IssueAssetStateFromStateRef
+import com.cordaSimpleApplication.state.AssetState
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.requireObject
+import com.github.ajalt.clikt.parameters.arguments.argument
+import net.corda.core.identity.CordaX500Name
+import java.lang.Exception
+import net.corda.core.messaging.startFlow
+
+/**
+ * The CLI command used to trigger a CreateState flow.
+ *
+ * @property quantity The numberOfTokens for the [AssetState].
+ * @property tokenType The tokenType for the [AssetState].
+ */
+class IssueAssetStateCommand : CliktCommand(help = "Invokes the IssueAssetState flow. Requires a key and value") {
+    private val quantity: String by argument()
+    private val tokenType: String by argument()
+    val config by requireObject<Map<String, String>>()
+    override fun run() {
+        issueAssetStateHelper(quantity.toInt(), tokenType, config)
+    }
+}
+
+/**
+ * Helper function used by IssueAssetStateCommand
+ */
+fun issueAssetStateHelper(numberOfTokens: Int, tokenType: String, config: Map<String, String>) {
+    val rpc = NodeRPCConnection(
+            host = config["CORDA_HOST"]!!,
+            username = "clientUser1",
+            password = "test",
+            rpcPort = config["CORDA_PORT"]!!.toInt())
+    try {
+        println("IssueAsset flow with arguments $numberOfTokens, $tokenType")
+        val proxy = rpc.proxy
+        val createdState = proxy.startFlow(::IssueAssetState, numberOfTokens, tokenType)
+                .returnValue.get().tx.outputStates.first() as AssetState
+        println(createdState)
+    } catch (e: Exception) {
+        println(e.toString())
+    } finally {
+        rpc.close()
+    }
+}
+
+class IssueAssetStateFromStateRefCommand : CliktCommand(help = "Invokes the IssueAssetStateFromStateRef flow. Requires a linearId") {
+    private val linearId: String by argument()
+    val config by requireObject<Map<String, String>>()
+    override fun run() {
+        val rpc = NodeRPCConnection(
+            host = config["CORDA_HOST"]!!,
+            username = "clientUser1",
+            password = "test",
+            rpcPort = config["CORDA_PORT"]!!.toInt())
+        try {
+            println("IssueAsset flow with arguments $linearId")
+            val proxy = rpc.proxy
+            val createdState = proxy.startFlow(::IssueAssetStateFromStateRef, linearId)
+                .returnValue.get().tx.outputStates.first() as AssetState
+            println(createdState)
+        } catch (e: Exception) {
+            println(e.toString())
+        } finally {
+            rpc.close()
+        }
+    }
+}
+
+/**
+ * The CLI command used to trigger a GetStateByTokenType flow.
```

Comment:
The name mentioned here doesn't match the function name below. Should it?

Reply:
The name mentioned here is the name of the flow called inside the function.